docs/pages/app-signing/apple-developer-program-roles-and-permissions.mdx

```diff
@@ -0,0 +1,55 @@
+---
+title: Apple Developer Program roles and permissions for EAS Build
+sidebar_title: Apple Developer Program roles and permissions
+description: Learn about the Apple Developer account membership requirements for creating an EAS Build.
+---
+
+import { Terminal } from '~/ui/components/Snippet';
+import { Step } from '~/ui/components/Step';
+
+An Apple Developer account is required to create iOS device builds on EAS. This account allows you to generate [app signing credentials](/app-signing/managed-credentials/#generating-app-signing-credentials) such as certificates, identifiers, and profiles, submit the app for review, and manage app's distribution.
+
+If the Apple Developer account is an individual account, only the account owner can generate the app signing credentials. This guide provides steps that the account owner has to follow to ensure app signing credentials are generated. It also provides steps for the team developer to create an EAS Build.
+
+> See [Apple's documentation on Program Roles](https://developer.apple.com/support/roles/) for details on the different roles and their permissions based on the type of Developer account.
+
+## Steps for Apple Developer account owner
+
+The owner of the individual Apple Developer account needs to generate the following credentials:
+
+- **Distribution signing certificate**: Required to sign development and release builds that are installed on an iOS device.
+- **Ad hoc provisioning profile**: Required to sign builds that are installed on a device outside of the Apple App Store.
+- **Distribution provisioning profile**: Required to sign the build that is submitted to the Apple App Store.
+- **Push key**: Required when using a push notification service.
+
+With EAS CLI, all of the above credentials can be created and synced automatically with the Apple Developer account. Once the owner login to their [Expo account](/accounts/account-types/), they can create or update the provisioning profile by running `eas credentials` using the EAS CLI.
+
+<Terminal cmd={['$ eas login', '', '$ eas credentials']} />
+
+The CLI will prompt for selecting a [build profile](/build/eas-json/#build-profiles) to use for the EAS Build. If the owner is creating a production build, follow the the steps to [create a distribution provisioning profile](/tutorial/eas/ios-production-build/#create-a-distribution-provisioning-profile). To create a developer build, follow the steps to [create a provisioning profile](/tutorial/eas/ios-development-build-for-devices/#provisioning-profile).
+
+This ensures that the provisioning profile associated with the Expo account has necessary permissions.
+
+> For projects with existing credentials, see [Using existing credentials](/app-signing/existing-credentials/) for details on how to sync these to EAS or manage them manually.
+
```

Comment:
```suggestion

### Uploading pre-generated Apple credentials
Some development teams may choose to generate distribution certificates and provisioning profiles outside of EAS. These credentials can be added by any EAS user with Developer or higher permissions via `eas credentials` or in **Credentials** under **Project Settings** on the Expo dashboard. You will need the **.p12** and **.mobileprovision** files, as well as any passwords set when generating the distribution certificate, when uploading the credentials.
```
Context: bigger teams will generate these on their own and distribute the files to those who need them because a) everyone having access to certificates, identifiers, and profiles can be a security concern, and b) you only get, I think, 3 distribution certs at once. These certs might be shared with EAS apps, non-RN apps, etc. So, someone going wild with generating credentials in EAS could grind the built process somewhere else in the organization to a halt.

So, I think it's good to mention somewhere that generating them via EAS isn't the only option. Also, what's nice here is that the individual apple account owner or otherwise responsible for Apple credentials person doesn't even need to be a member of your Expo organization (in one case, I was working with someone from a security team in some area of the org I never otherwise interacted with).

Reply:
I'm adding this as a sub section under **Additional information** as I'm not sure if this should come just after the introduction or something needs to be highlighted that early in the documentation.